bioimageio/core/prediction.py

```diff
@@ -288,6 +299,14 @@ def check_tiling(tiling):
             assert isinstance(shape, list)
             assert len(shape) == len(axes)
 
+            scale = None
+            output_shape = output_spec.shape
+            if not isinstance(output_shape, list):
+                scale = output_shape.scale
+            else:
+                scale = [1.0] * len(axes)
```

Comment:
In this case we have a hard-coded output shape. I think it's not clear that this would correspond to having `scale=1.0`.
I would rather throw a NotImplementedError here.

Reply:
Fixed. I put an assert at the top so that tiling will always fail if the output_shape is provided as a simple list rather than an implicit shape. This is a breaking change in the cases where scale was not provided, but input_shape and output_shape were, with output_shape being a simple list.
It makes sense to me that tiling should not work if the output shape is simply provided as a list.